catroid/src/main/java/org/catrobat/catroid/content/Project.java

```diff
@@ -279,7 +296,7 @@ public XmlHeader getXmlHeader() {
 	}
 
 	public int getRequiredResources() {
-		int resources = Brick.NO_RESOURCES;
+		int resources = ProjectManager.getInstance().getCurrentProject().isCastProject() ? Brick.CAST_REQUIRED : Brick.NO_RESOURCES;
```

Comment:
- access project without digging through projectManager
- initialize resources still to NO_RESOURCES, and then add CAST_REQUIRED if needed in a separate line